### ISSUE TITLE
[feature reques] gitlab_label support for groups

### ISSUE THREAD

**Issue by ketzacoatl** [terraform-providers/terraform-provider-gitlab#155]
Hi there,

This is a feature request.

### Terraform Version

`v0.12.2`.

### Affected Resource(s)

`gitlab_label`

### Terraform Configuration Files
```hcl
resource "gitlab_label" "done" {
  name        = "Done"
  description = "This task is complete."
  group     = "${gitlab_group.my-group.id}"
  color       = local.label_color["done"]
}

```

### Expected Behavior

`gitlab_label` should let you create a label that is associated with a group, and not a project.

### Actual Behavior

`gitlab_label` requires `project` and does not support a `group` attribute.

Thanks!

**Comment by beanaroo** [comment]
Good news! It looks like `gitlab_group_label` has been merged in https://github.com/terraform-providers/terraform-provider-gitlab/pull/186

Hopefully we'll see it in the next release

**Comment by ketzacoatl** [comment]
Awesome! What is the release schedule like? Seems like "not frequent". Is there a technical limitation for that? Should I just build/install the provider from a specific commit?

**Comment by roidelapluie** [comment]
It is released in 2.4.0
